api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -484,6 +484,7 @@ public void createApp(CreateAppRequest createAppRequest, DbWorkspace dbWorkspace
                   .collect(Collectors.toList()));
       List<Disk> appDisks =
           diskList.stream()
+              .filter(d -> d.getAppType() != null)
```

Comment:
is the appType `null` because the disk is associated with a Jupyter runtime instead of a GKE App?  If so, please add a comment explaining this.

Might be better to move this line to L483 as well, to catch nulls as early as possible